apps/src/code-studio/components/LabContainer.jsx

```diff
@@ -11,19 +11,42 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import {useSelector} from 'react-redux';
+import classNames from 'classnames';
 import moduleStyles from './LabContainer.module.scss';
+import i18n from '@cdo/locale';
 
 const LabContainer = ({children}) => {
   const currentLevelId = useSelector(state => state.progress.currentLevelId);
```

Comment:
Do we still need to force a re-render based on currentLevelId now that we have the isLabLoading flag?

Reply:
Good call, apparently not!